srcpkgs/cereal/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'cereal'
+pkgname=cereal
+version=1.3.0
+revision=1
+archs=noarch
+wrksrc=cereal-${version}
+build_style=cmake
+configure_args="-DSKIP_PORTABILITY_TEST=1 -DTHREAD_SAFE=ON -DWITH_WERROR=OFF"
+short_desc="Header-only C++ serialization library"
+maintainer="Orphaned <orphan@voidlinux.org>"
```

Comment:
New packages should not be Orphaned

Reply:
suggestion ? don't want to put my e-mail in there

Reply:
Why not? You already have your email in this commit.